sdk/spring/azure-identity-spring/src/main/java/com/azure/spring/identity/DefaultSpringCredentialBuilder.java

```diff
@@ -15,54 +26,52 @@
  */
 public class DefaultSpringCredentialBuilder extends SpringCredentialBuilderBase<DefaultSpringCredentialBuilder> {
 
-    /**
-     * Defines the AZURE_CREDENTIAL_PREFIX.
-     */
-    static final String AZURE_CREDENTIAL_PREFIX = "azure.credential.";
+    private String managedIdentityClientId;
+    private String tenantId;
 
-    private String alternativePrefix;
+    public DefaultSpringCredentialBuilder(Environment environment) {
```

Comment:
Builders like this didn't bring much benefits, how about deleting these builders and use constructor instead?

Reply:
just followed the sdk builder pattern.